variables.tf

```diff
@@ -1,5 +1,11 @@
-variable "length" {
-  description = "The length of the random name"
-  type        = number
-  default     = 2
+variable "users" {
+  type = list(object({
+    access_roles         = map(any)
```

Comment:
I'm sorry, this is me being painful and not clear: I don't want this to be prescriptive to what we've done with clients or our structure elsewhere. If this is open source, then it should be as natural as possible. And therefore instead of `access_roles` how about we just use the `role` attribute so that the whole structure will just be: 

```hcl
{
    email    = "jane.smith@example.com",
    name     = "Jane Smith",
    role     = "read_only",
    username = "janesmith"
  disabled = false
  }
```

Reply:
And we should validate that role is one of the 3 enum values. Cause otherwise 💥

Reply:
Updated. And I realized that the roles should be a list, so this is also updated.